packages/dm-core-plugins/src/file/UploadFileButton.tsx

```diff
@@ -1,23 +1,26 @@
-import { TLinkReference, useDMSS } from '@development-framework/dm-core'
+import {
+  splitAddress,
+  TLinkReference,
+  TStorageReference,
+  useDMSS,
+} from '@development-framework/dm-core'
 import { Button, Progress } from '@equinor/eds-core-react'
 import { AxiosError } from 'axios'
 import * as React from 'react'
 import { ChangeEvent, useEffect, useRef, useState } from 'react'
-import { useRegistryContext } from '../form/context/RegistryContext'
```

Comment:
using a registry from form plugin in the file plugin does not make sense... therefore It is removed and props to component are updated

Reply:
Wouldn't that throw the `useRegistryContext must be used within a RegistryProvider` warning?

Reply:
it didnt seem to give any errors, but the value recieved from useRegistryContext() was sometimes empty